pages/apis/rest_api.md

```diff
@@ -60,7 +60,11 @@ curl -H "Authorization: Bearer $TOKEN" https://api.buildkite.com/v2/user
 
 ## Pagination
 
-For endpoints which support pagination, the pagination information can be found in the `Link` HTTP response header containing zero or more of `next`, `prev`, `first` and `last`.
+For endpoints that support pagination, the pagination information can be found in the `Link` HTTP response header.
+
+### Page number pagination
+
+Unless specified otherwise, endpoints are paginated using page numbers, with the `Link` header containing zero or more of `next`, `prev`, `first` and `last`.
 
```

Comment:
I think it would be good to add a sentence to introduce the example. Perhaps something like:

```suggestion

For example, the following request and response to get a list of builds for a pipeline contains page number details for `next` and `last`:

```

What do you think?